protobuf/model_config.proto

```diff
@@ -1221,11 +1223,21 @@ message ModelSequenceBatching
     //@@
     repeated float fp32_false_true = 3;
 
+    //@@    .. cpp:var:: bool bool_false_true (repeated)
+    //@@
+    //@@       The control's true and false setting is indicated by setting
+    //@@       a value in a bool tensor. The tensor must be a
+    //@@       1-dimensional tensor with size equal to the batch size of
+    //@@       the request. 'bool_false_true' must have two entries: the
+    //@@       first the false value and the second the true value.
+    //@@
+    repeated float bool_false_true = 4;
+
     //@@    .. cpp:var:: DataType data_type
     //@@
     //@@       The control's datatype.
     //@@
-    DataType data_type = 4;
```

Comment:
For backwards compatibility for the protobuf files should not change the identifier of the fields. Leave `data_type = 4` and make `bool_false_true = 5`

> you must not change the tag numbers of any existing fields.

Source: https://developers.google.com/protocol-buffers/docs/pythontutorial#extending-a-protocol-buffer

Reply:
Thanks for the link. Updated.